docs/ReleaseNotes.md

```diff
@@ -17,7 +17,7 @@ Starting with version [3.4.455.0](#344550), the semantics of `UnnestedRecordType
 // begin next release
 ### NEXT_RELEASE
 
-* **Bug fix** Fix 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Bug fix** The `ArithmeticValue` and `NumericAggregationValue` classes now uses a fixed `Locale` with `toUpperCase` when encapsulating a specified function by name to avoid mismatches when running in the Turkish locale [(Issue #2750)](https://github.com/FoundationDB/fdb-record-layer/issues/2750)
```

Comment:
These changes are highlighted because it's the only part that should be API-visible (in that if someone was running with a Turkish locale and specified the function name `"multıply"`, they'll need to change it to `"multiply"`, but it seems unlikely that someone would intentionally do the former). The rest of the changes should only affect logging or tests